plugin-locationcomponent/src/test/java/com/mapbox/maps/plugin/locationcomponent/animators/PuckAnimatorManagerTest.kt

```diff
@@ -99,7 +98,7 @@ class PuckAnimatorManagerTest {
       bearingAnimator.animate(0.0, 10.0, options = options)
     }
     Shadows.shadowOf(Looper.getMainLooper()).idle()
-    MatcherAssert.assertThat(counter, Matchers.greaterThan(0))
+    assertTrue(counter > 0)
```

Comment:
🤔 Indeed it looks easier... idk why I made it that complicated

Reply:
We're also using similar `MatcherAssert` in several other places, can it be also replaced? Then we can remove `hamcrest` dependency at all from modules

Reply:
this one for some reason was having trouble finding that import. Fine for now, we can look more into removing others as tailwork.